### ISSUE TITLE
k-fold update

### ISSUE THREAD

**Issue by georgemundo** [lmudeeplearninglab/Autopipe#11]
increase number of epochs, and shuffle values in kfold training and validation setes.

**Comment by EARound** [comment]
ran k fold value of 10 with 100 epochs.  overfitting began in the teens and results were poor.  run log attached for reference.  Uncropped image for full image processing in CNN.  if better preprocessing is implemented then results may improve.
[testout4.txt](https://github.com/lmudeeplearninglab/Autopipe/files/3155366/testout4.txt)

**Comment by EARound** [comment]
should end weights consist of an average of the separate k fold model weights?

**Comment by EARound** [comment]
Kernel size
Fc size
Epoch number - attempted 100, overfitting observed in epoch number X-teen
Dropout value

**Comment by EARound** [comment]
performed assessment with 25 epochs on 2/3 of image.  loss minimizes roughly around 12 epochs.  will run additional tests for kernel size manipulation and dropout impacts.  defer changing quantity of FC layers and will follow up with implementing cropping techniques.
[testout5.txt](https://github.com/lmudeeplearninglab/Autopipe/files/3156269/testout5.txt)

**Comment by EARound** [comment]
final assessment:

lessons learned: cv2.resize requires input tuple defined by [dx,dy] which is inconsistent with other image definitions of [rows, cols].  Not enough data to make strong statements.  training, validating, and testing sets lacked variation.

The code i am uploading has changed the took by giving more variability.  the k-fold has been properly implemented and the weights are dynamically called.  Results were poor, however consistent.  

settings to be reviewed for further study: model layers, dropout layer percentages (commented out second dropout layer for data provided), epochs, k-folds, percentage or test data, end image size, preprocessing, and postprocessing.  

Closing issue by merging my branch with the trunk.


[testout6.txt](https://github.com/lmudeeplearninglab/Autopipe/files/3168796/testout6.txt)
